src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
```

Comment:
Cool, will update the comment, but it doesn't answer my question, those situations are happen sometimes, most recently in https://github.com/jquery/jquery/commit/a5e1c9b44c971fd7046d9a95bd0810e50840b663.

In that case, we know problem exist in Chrome 34, but it might also be present in 33, but there is not actual reason to check them all, but we can very well mislead the person who would read such comment, perhaps we need "at least" notation for these kind of stoppers?